scripts/qiita-private-launcher

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from subprocess import Popen, PIPE
+from os import close, remove
+from tempfile import mkstemp
+
+import click
+
+# When Popen executes, the shell is not in interactive mode,
+# so it is not sourcing any of the bash configuration files
+# We need to source it so the env_script are available
```

Comment:
Yeah, it is hidden in the documentation and if you don't pay full attention it is really hard to realize that. I guess that is a result on how the bashrc and bash_profile are designed to work, but I don't think everybody uses them correctly. Furthermore, Mac is the exception and they don't follow those rules... fun fun!